feat: Member room IDs read bridge (#37057)
RocketChat/Rocket.Chat: .changeset/chatty-foxes-attend.md

---
'@rocket.chat/apps-engine': minor
'@rocket.chat/meteor': minor
---

> Adds a user room ID bridge to the apps engine so apps can retrieve members’ room identifiers.

Reviewer: ```suggestion
Adds a user bridge method to apps engine so apps can retrieve the room ids the user is part of. 
```
